### DISCUSSION TITLE
Feature Request: Support for OP Sqlite

### DISCUSSION THREAD

**Discussion by gokul1099** [electric-sql/electric#882] [Feature requests]
opsqlite has much better performance benchmarks compare to expo-sqlite and other sql libraries. it would be great if electric supports opsqlite.
https://github.com/OP-Engineering/op-sqlite/tree/main

**Comment by linear** [comment]
<p><a href="https://linear.app/electric-sql/issue/VAX-1536/feature-request-support-for-op-sqlite">VAX-1536 Feature Request: Support for OP Sqlite</a></p>
